backend/serializers/hacknight_serializer.py

```diff
@@ -9,3 +9,4 @@ class Meta:
     participants = fields.Nested(
         "ParticipantSchema", exclude=("hacknights",), many=True
     )
+    date = fields.Date()
```

Comment:
What happens here?

Reply:
Serialization from date js object to python date format